pycqed/analysis_v3/data_processing.py

```diff
@@ -1,232 +1,104 @@
 import logging
 log = logging.getLogger()
 log.addHandler(logging.StreamHandler())
```

Comment:
Loggers should have a name, ie `__name__`. Creating several of  those loggers without name (ie. root loggers) might be responsible for our log duplicates (but I m not sure about that).